handler/handler.go

```diff
@@ -16,33 +15,33 @@ type ReindexRequestedHandler struct {
 	Config     config.Config
 }
 
-type key int
-
-const keyRequestID key = iota
-
 // Handle takes a single event.
 func (h *ReindexRequestedHandler) Handle(ctx context.Context, event *models.ReindexRequested) error {
 	logData := log.Data{
 		"event": event,
 	}
-	ctx = context.WithValue(ctx, keyRequestID, event.TraceID)
+
 	log.Info(ctx, "reindex requested event handler called", logData)
 
 	publishedIndex, err := h.ZebedeeCli.GetPublishedIndex(ctx, nil)
 	if err != nil {
+		// the cfg.ZebedeeClientTimeout maybe needs to be increased
```

Comment:
Comment seems out of place if not, why does cfg.ZebedeeClientTimeout needs to be increased? Possible add a little more context and start the comment with TODO or investigate. Also create a new ticket for this investigation?

Reply:
Yeah a bit more info would be useful here, no need to investigate though. The endpoint on Zebedee we are calling is to retrieve all published files on disk, this operation in Zebedee takes a long time to process and respond to caller (search data finder). Problem is that as more and more data/content is published, more data is stored on disk in Zebedee and hence the time to process will increase.

The solution would be to add pagination onto this endpoint but this is blocked by the fact the data is not stored in a database. Would also need a sort mechanism by release date to prevent new docs being added while paging through resources.

Reply:
Also worth noting if Zebedee is processing other requests, or one long request, it may not handle this request in good time. Problem with there being 1 instance of Zebedee but again limited to only 1 in publishing because working with files on disk.